File: dogmeet-projekt/dogMeet_laravel/app/Models/Dog.php
<?php

namespace App\Models;

use Illuminate\Database\Eloquent\Factories\HasFactory;
use Illuminate\Database\Eloquent\Model;

class Dog extends Model
{
    use HasFactory;

    protected $table = "dogs";
    public $timestamps = false;
    protected $fillable = [
        "name",
        "age",
        "type",
        "gender",
        "description",
        "owner_id"
    ];

    public function user()
    {
        return $this->belongsTo(User::class, "owner_id", "id");
    }

    public function pictures()
    {
        return $this->hasOne(DogPictures::class, 'dog_id', 'id');
    }

    public function histories()
    {
        return $this->hasMany(DogHistory::class);
    }
}

File: dogmeet-projekt/dogMeet_admin/dogMeet_adminSite/dogMeet_adminSite/Model/Dog.cs

namespace dogMeet_adminSite.Model
{
    public class Dog
    {
        public int id { get; set; }
        public string name { get; set; }
        public int age { get; set; }
        public string type { get; set; }
        public string gender { get; set; }
        public string description { get; set; }
        public int? owner_id { get; set; }

        public Dog() { }
    }
}
